src/schema/v2/RenameFields.ts

```diff
@@ -152,7 +153,12 @@ function fieldKey(type: TypeWithSelectableFields, fieldName: string) {
 function getTypeWithSelectableFields(
   typeInfo: TypeInfo
 ): TypeWithSelectableFields {
-  const type = getNamedType(typeInfo.getType())
+  let type
+  if (typeInfo.getType() instanceof GraphQLList) {
+    return getNamedType(typeInfo.getParentType())
+  } else {
+    type = getNamedType(typeInfo.getType())
+  }
```

Comment:
I think it might read better if `typeInfo.getType()` or the parent is done using the ternary operator.

Reply:
the thing is, the true case of `if` ends up early returning, i did some updates hopefully this makes it more readable.